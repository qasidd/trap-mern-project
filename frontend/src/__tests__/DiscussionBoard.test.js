import Discussion from '../components/pages/Discussionboard';
import '@testing-library/jest-dom';
import { expect, it } from '@jest/globals';
import { create } from 'react-test-renderer';

it(`It should match the snapshot every time it's rendered`, () => {
    const instance = create(<Discussion/>).toJSON();
    expect(instance).toMatchSnapshot();
    
})