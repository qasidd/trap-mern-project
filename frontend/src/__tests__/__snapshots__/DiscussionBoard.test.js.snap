// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should match the snapshot every time it's rendered 1`] = `
<div
  className="row"
>
  <div
    className="col-md-3"
  >
    <div
      className="bg-light border-right"
      id="sidebar"
    >
      <div
        className="card"
      >
        <div
          className="card-body"
        >
          <div
            className="card-title"
          >
            Create New Post
          </div>
          <form
            onSubmit={[Function]}
          >
            <input
              className="form-control"
              onChange={[Function]}
              placeholder="Username"
              type="text"
              value=""
            />
            <input
              className="form-control"
              onChange={[Function]}
              placeholder="Movietitle"
              type="text"
              value=""
            />
            <input
              className="form-control"
              onChange={[Function]}
              placeholder="Comment"
              type="text"
              value=""
            />
            <input
              className="form-control"
              onChange={[Function]}
              placeholder="Rating"
              type="number"
              value={0}
            />
            <input
              className="form-control"
              onChange={[Function]}
              placeholder="Created"
              type="date"
              value=""
            />
            <br />
            <button
              className="btn btn-outline-success col-md-12"
              type="submit"
            >
              Post
            </button>
          </form>
        </div>
      </div>
    </div>
  </div>
  <div
    className="container"
  >
    <div
      className="col-md-9"
    >
      <div
        className="alert alert-success"
      >
        
      </div>
      <div
        animation="border"
        className="spinner-border"
        role="status"
      >
        <span
          className="sr-only"
        >
          Loading...
        </span>
      </div>
    </div>
  </div>
</div>
`;
